integration/v7/isolated/service_command_test.go

```diff
@@ -171,17 +171,16 @@ var _ = Describe("service command", func() {
 				})
 
 				It("displays the bound apps", func() {
-					//Delay to reduce flakiness
-					time.Sleep(10 * time.Second)
-					session := helpers.CF(serviceCommand, serviceInstanceName, "-v")
-					Eventually(session).Should(Exit(0))
-
-					Expect(session).To(SatisfyAll(
-						Say(`Showing bound apps:\n`),
-						Say(`name\s+binding name\s+status\s+message\n`),
-						Say(`%s\s+%s\s+create succeeded\s*\n`, appName1, bindingName1),
-						Say(`%s\s+%s\s+create succeeded\s*\n`, appName2, bindingName2),
-					))
+					Eventually(func(g Gomega) {
+						session := helpers.CF(serviceCommand, serviceInstanceName, "-v")
+						g.Eventually(session).Should(Exit(0))
```

Comment:
```suggestion
						g.Expect(session).Should(Exit(0))
```

Reply:
Eventually blocks when called and attempts an assertion periodically until it passes or a timeout occurs. We can use Expect here and rely on Eventually on line 174 to retry